envoy-control-tests/src/main/resources/lua_spec/ingress_client_name_header_spec.lua

```diff
@@ -0,0 +1,114 @@
+require("ingress_client_name_header")
+
+local function handlerMock(headers, metadata, https, uri_san_peer_certificate)
+    local downstreamSslConnection_mock = mock({
+        uriSanPeerCertificate = function() return uri_san_peer_certificate end
+    })
+    return {
+        headers = function() return {
+            add = function(_, key, value) if headers[key] ~= nil then headers[key] = headers[key] ..","..value else headers[key] = value end end,
+            get = function(_, key)
```

Comment:
Do we care that it's duplicated?

Reply:
Could you explain what do you mean?

Reply:
I was thinking of marking a common factory of handlerMocks, [this](https://github.com/allegro/envoy-control/blob/get-client-name-from-uri-san-peer-cert/envoy-control-tests/src/main/resources/lua_spec/ingress_rbac_logging_spec.lua#L9-L38) and [this](https://github.com/allegro/envoy-control/blob/get-client-name-from-uri-san-peer-cert/envoy-control-tests/src/main/resources/lua_spec/ingress_client_name_header_spec.lua#L3-L29) are pretty much the same.

Reply:
I would do that, but I have problems with using busted functions, like `mock` in imported file